File: src/codes/Users_Manual/gain/transform_0.rs
# extern crate autd3;
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
let g = Uniform::new(EmitIntensity::MAX).with_transform(|_dev| |_tr, d| Drive::new(
    Phase::from((d.phase().radian() + PI) * rad),
    d.intensity() / 2,
));
# }

File: src/codes/Users_Manual/gain/transform_0.py
~import numpy as np
~from pyautd3 import Drive, EmitIntensity, Phase, rad, Uniform
g = Uniform(EmitIntensity.maximum()).with_transform(
    lambda dev: lambda tr, d: Drive(
        Phase((d.phase.radian + np.pi) * rad), d.intensity // 2
    )
)